events/wc_dark_cult_events.txt

```diff
@@ -1114,6 +1170,232 @@ narrative_event = {
 				character_event = { id = MNM.10030 }
 			}
 			narrative_event = { id = WCDAC.5050 }
+			character_event = { id = WCDAC.5041 days = 7 }
+			character_event = { id = WCDAC.5045 years = 5 }
+		}
+	}
+}
+character_event = {
+	id = WCDAC.5041
```

Comment:
Garona killed Llane... I think it's more or less lore-friendly, but he's definitely not my follower.
<details>
<summary>Click to expand</summary>

![image](https://user-images.githubusercontent.com/46384992/79261259-9f422200-7ea0-11ea-9704-b2b66a2b94bd.png)

</details>
P.S. I can get a lot of these screenshots (Garona is my wife, Garona is Old Gods' follower), but it takes a lot of time, so just rework these triggers. There are so many obvious ways to break them.